iFixit/react-commerce
# Add main category and category to blocklist

fixes #191 

## Changelog

- Add `Main Category` and `Category` to facet blocklist

## QA

1. Visit the Vercel PR preview (replace `vercel.app` with `cominor.com`)
2. Visit the parts product list
3. Verify that neither `Main Category`, nor `Category` are displayed in the facets list
